Content.Client/Content.Client.csproj

```diff
@@ -23,6 +23,9 @@
     <ProjectReference Include="..\RobustToolbox\Robust.Client\Robust.Client.csproj" />
     <ProjectReference Include="..\Content.Shared\Content.Shared.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="_Moffstation\Commands\" />
+  </ItemGroup>
```

Comment:
```suggestion
```
I love Rider

edit: or maybe I should sarcastically say "I love MSBuild", I'm not sure which is the true villain here.